vignettes/documentation.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
-title: "eventstudyr"
+title: "documentation"
```

Comment:
@santiagohermo was what your reasoning for changing the title? For the `dplyr` vignette, the name "dplyr" is still in the [vignette's title](https://github.com/tidyverse/dplyr/blob/29307bf661a8c52b892c28afa531b38461aababb/vignettes/dplyr.Rmd#L2)

Reply:
I think it was changed automatically when I created the vignette `usethis::use_vignette("documentation")`. I just copied the content, without altering the `yaml`.

I'm fine with the new title as long as the vignette still ships!

Reply:
I don't have strong feelings about the vignette title so this is okay with me, marking as resolved. Thanks!